versions/3.0.4.md

```diff
@@ -3523,3 +3588,7 @@ Version   | Date       | Notes
 1.2       | 2014-03-14 | Initial release of the formal document.
 1.1       | 2012-08-22 | Release of Swagger 1.1
 1.0       | 2011-08-10 | First release of the Swagger Specification
+
+## <a name="usingRFC6570Implementations"></a>Appendix C: Using RFC6570 Implementations
```

Comment:
Do these empty sections indicate that this pull request is still work-in-progress?Perhaps the content could go into an external resource rather than into the spec?

Reply:
Empty sections are there to placate the PR checks that complain if you link to a section that isn't there.

All sections are present in one PR or another right now.  I didn't know how else to do this.